packages/react-aria-components/src/Form.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {FormValidationContext} from '@react-stately/form';
+import React, {FormHTMLAttributes, ForwardedRef, forwardRef} from 'react';
+import {ValidationErrors} from '@react-types/shared';
+
+export interface FormProps extends FormHTMLAttributes<HTMLFormElement> {
+  /**
+   * Validation errors for the form, typically returned by a server.
+   * This should be set to an object mapping from input names to errors.
+   */
+  validationErrors?: ValidationErrors | null
+}
+
+function Form(props: FormProps, ref: ForwardedRef<HTMLFormElement>) {
```

Comment:
Description says small wrapper and can add in the future, so I assume suggestions like providing data attributes and renderprops for focusWithin or a FormContext for passing in props would fall into that category of enhancements?

Reply:
good questions. I'm not sure what the use case for focus within on a form would be. I thought our recommendation was to use Group in those cases. I guess we could support FormContext for consistency but not really sure what people would use it for.

Reply:
Yeah, I realized you could just use a Group, though don't know if it'd be a nice convenience feature

Reply:
We do very explicitly say
> All React Aria Components export a corresponding context that can be used to send props to them from a parent element

in our docs, just noticed